File: CheckMyStar.Bll.Requests/CriterionUpdateRequest.cs
using CheckMyStar.Bll.Models;

namespace CheckMyStar.Bll.Requests
{
    public class CriterionUpdateRequest
    {
        public StarCriterionModel? Criterion { get; set; }
        public StarLevelModel? StarLevel { get; set; }
        public StarLevelCriterionModel? StarLevelCriterion { get; set; }
    }
}

File: CheckMyStar.Angular/src/app/www/40_Requests/BackOffice/Criterion-update.request.ts
import { StarCriterionModel } from '../../20_Models/BackOffice/Criterion.model';
import { StarLevelModel } from '../../20_Models/BackOffice/StarLevel.model';
import { StarLevelCriterionModel } from '../../20_Models/BackOffice/StarLevelCriterion.model';

export interface CriterionUpdateRequest {
  criterion : StarCriterionModel;
  starLevel: StarLevelModel;
  starLevelCriterion: StarLevelCriterionModel;
}